integration-tests/opentelemetry/src/test/java/io/quarkus/it/opentelemetry/OpenTelemetryTestCase.java

```diff
@@ -303,7 +303,7 @@ void testSlashClientPath() {
                 Assertions.assertEquals("/", spanData.get("attr_http.target"));
                 Assertions.assertEquals(pathParamUrl.getAuthority(), spanData.get("attr_http.host"));
                 Assertions.assertEquals("http", spanData.get("attr_http.scheme"));
-                Assertions.assertNull(spanData.get("attr_http.route"));
+                Assertions.assertEquals("/", spanData.get("attr_http.target"));
```

Comment:
This changed from testing `route` to `target`, which is already tested on line 303

Reply:
Correct. Wrong copy paste.